README.md

```diff
@@ -14,23 +14,109 @@
   </a>
 </a>
 </p>
+
+<h2 align="center"><a href="https://killercoda.com/open-feature/scenario/flagd-demo" target="_blank">Try flagd in the Killercoda Playground (in browser)</a></h4>
 <!-- markdownlint-enable MD033 -->
 
 ## Features
 
+Flagd is a simple command line tool for fetching and evaluating feature flags for services.
+
+Think of it as a ready-made, open source, OpenFeature compliant feature flag backend system.
+
 - OpenFeature compliant with providers available in many languages
-- Multiple flag configuration sources including `files`, `http`, and `Kubernetes`
+- Multiple flag configuration sources, including `files`, `http`, `gRPC` and `Kubernetes` can be used simultaneously
```

Comment:
```suggestion
- Multiple flag configuration sources, including `files`, `http`, `gRPC` and `Kubernetes`
```

I don't think you need that section because it's listed below.